### PR TITLE
Fix tests namespace

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

File: test/Microsoft.AspNetCore.Localization.Routing.Tests/RouteDataRequestCultureProviderTest.cs [hunk 1]
```diff
@@ -8,12 +8,13 @@
...
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Localization.Routing;
 using Microsoft.AspNetCore.Routing;
 using Microsoft.AspNetCore.TestHost;
 using Microsoft.Extensions.DependencyInjection;
 using Xunit;
 
-namespace Microsoft.AspNetCore.Localization.Routing
+namespace Microsoft.AspNetCore.Localization.Routing.Tests
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Localization.Routing.Tests/RouteDataRequestCultureProviderTest.cs]: This one's fins as it is. Our test namespaces have the same namespace as the type being tested.

File: test/Microsoft.Extensions.Localization.Tests/LocalizationServiceCollectionExtensionsTest.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@
 using Microsoft.Extensions.Options;
 using Xunit;
 
-namespace Microsoft.Extensions.Localization.Test
+namespace Microsoft.Extensions.Localization.Tests
```
Comment (pranavkm) [inline test/Microsoft.Extensions.Localization.Tests/LocalizationServiceCollectionExtensionsTest.cs]: This needs to be `Microsoft.Extensions.DependencyInjection`

File: test/Microsoft.AspNetCore.Localization.Routing.Tests/RouteDataRequestCultureProviderTest.cs (hunk 1 above)
Comment (hishamco) [inline test/Microsoft.AspNetCore.Localization.Routing.Tests/RouteDataRequestCultureProviderTest.cs]: If you have the other test projects you will notice that all of them ends with `Tests` namespace

File: test/Microsoft.Extensions.Localization.Tests/LocalizationServiceCollectionExtensionsTest.cs (hunk 2 above)
Comment (hishamco) [inline test/Microsoft.Extensions.Localization.Tests/LocalizationServiceCollectionExtensionsTest.cs]: As it's part of `Microsoft.Extensions.DependencyInjection`

**Comment by Eilon** [comment]
@pranavkm can you do the needful?

**Comment by pranavkm** [comment]
We'll use https://github.com/aspnet/Localization/pull/400 to cover this. There's a bunch of incorrect test namespaces in this repo.

**Comment by hishamco** [comment]
Ok

**Pull request closed without merging**